communities/plymouth-university/community-assessment.md

```diff
@@ -0,0 +1,122 @@
+# Plymouth University
+
+Plymouth University has been having troubles with the tech community for a number of years. We've seen a seady decline in the number of tech based societies and events since around 2012. We are now in a situation where the last community left is the Game Dev Society (GDSoc). Thankfully there are still a few pockets of students who are enthusiastic about various strands of tech.
+
+I'll be using this assessment to investigate the culture and community of Plymouth University as a whole rather than focusing on the GDSoc alone.
+
+## Definition
+
+Students at the University of Plymouth, UK (known as Plymouth University) who have an interest in technology and computer science. These students may be on computing / computer science courses or just have an interest they're looking to explore.
+
+## Size
+
+### How many people are currently in the community?
+
+The size of the current involved community is sitting around **50**. This is just the engaged community who have together to take part and create events for themselves, primarily this is GDSoc along with a number of enthusiastic students.
+
+### How many would you _like_ to be in the community?
+
+In the distant future it would be wonderful to see a of the majority of the School of Computing engaging in the community, this would be around 200.
+
+In a smaller time frame it would be good to get all 50 together for regular meets such as workshops and talks.
+
+### Historically, has the group always been about the same size? If not, what brought on that change?
+
+The University had a lot more tech activity when I was a fresher in 2012. It seems that students are less willing to turn up to events now, the quality seems to have remained the same and even improved yet turn out is much lower now. There is a lot less "buzz" around tech news, new tools, and opportunities.
```

Comment:
This is interesting. Dealing with students' apathy to go to events is one of the toughest challenges a school community face. It could be usefull to know why the people that used to attend events stopped. Reaching freshmen is a great way to get people into events!